apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/AccessJsonE2E.java

```diff
@@ -0,0 +1,123 @@
+package gov.cms.bfd.server.war;
+
+import static io.restassured.RestAssured.given;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import gov.cms.bfd.model.rif.entities.Beneficiary;
+import gov.cms.bfd.server.sharedutils.BfdMDC;
+import gov.cms.bfd.server.war.commons.TransformerConstants;
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/** Verifies that access.json is written to as expected. */
+public class AccessJsonE2E extends ServerRequiredTest {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(AccessJsonE2E.class);
+
+  /** Verifies that access.json is written to within BFD-server-war via API call. */
+  @Test
+  public void verifyAccessJson() throws IOException {
+
+    Beneficiary beneficiary = testUtils.getFirstBeneficiary(testUtils.loadSampleAData());
+    String mbiHash = beneficiary.getMbiHash().orElseThrow();
+    String requestString =
+        baseServerUrl
+            + "/v2/fhir/Patient/?identifier="
+            + TransformerConstants.CODING_BBAPI_BENE_MBI_HASH
+            + "|"
+            + mbiHash;
+
+    Path accessLogJson =
+        ServerTestUtils.getWarProjectDirectory()
+            .resolve("target")
+            .resolve("server-work")
+            .resolve("access.json");
+
+    // Empty the access json to avoid pollution from other tests
+    try (BufferedWriter writer = Files.newBufferedWriter(accessLogJson)) {
+      writer.write("");
+      writer.flush();
```

Comment:
This would do the same thing in one line:

`Files.writeString(accessLogJson, "")`